GOVERNANCE.md

```diff
@@ -4,51 +4,170 @@ This document defines governance policies for the Kiali project.
+* ability to collaborate with the team,
+* ability to document testing procedures of Kiali features and to update existing documentation if features change,
+* understanding of team policies and processes.
+
+You can propose yourself to apply for the Tester role, or a Maintainer can do it for you.
+Proposals must be posted by opening a pull request
+modifying the testers list to add the new proposed tester, or via a in
+a [GitHub discussion](https://github.com/kiali/kiali/discussions/new) under the
+Governance category. The following information must be provided:
+
+* nominee's GitHub username
+* a list of pull requests (top 5) tested by the nominee.
+* a list of documented test cases (if any)
+
+Formalization of the Tester role is granted if a simple majority vote
+among current maintainers succeeds.
+
+## Leaders
+
+Kiali leaders are Maintainers who have a broad knowledge about the Kiali project, its goals and vision.
+They may also be aware on how the ecosystem related to Kiali is evolving, and may also be aware of related projects.
+Thus, they are able to guide and mentor other Maintainers, give direction, and set priorities to the Kiali project.
+
+Current list of leaders (alphabetically ordered):
+
+* [abonas](https://github.com/abonas) - _Owner_
+* [jshaughn](https://github.com/jshaughn) - _Owner_
+* [lucasponce](https://github.com/lucasponce) - _Owner_
+
+### Becoming a Leader
+
+To become a Leader, you must first be a Maintainer. Then, a new Leader must be proposed
+by an existing Maintainer by opening a pull request modifying the leaders list to add the new
+proposed leader, or via a by opening a [GitHub discussion](https://github.com/kiali/kiali/discussions/new)
+under the Governance category. The following information must be provided:
+
+* nominee's GitHub username,
+* an explanation of why the nominee should be a leader.
+
+Leadership is granted if a simple majority vote among current maintainers succeeds.
+
+## Inactivity
+
+It is important for contributors to be and stay active to set an example and show commitment to the project. Inactivity is harmful to the project as it may lead to unexpected delays, contributor attrition, and a lost of trust in the project.
+
+* Inactivity is measured by:
+    * Periods of no contributions for longer than 4 months
+    * Periods of no communication for longer than 4 months
+* Consequences of being inactive include:
+    * Involuntary removal or demotion
+    * Being asked to move to Emeritus status
+
+## Involuntary Removal or Demotion
+
+Involuntary removal/demotion of a contributor happens when requirements to be on certain role aren't being met. This may include repeated patterns of inactivity, extended period of inactivity, a period of failing to meet the requirements of your role, and/or a violation of the Code of Conduct. This process is important because it protects the community and its deliverables while also opens up opportunities for new contributors to step in.
```

Comment:
```suggestion
Involuntary removal/demotion of a contributor happens when requirements for a certain role aren't being met. This may include repeated patterns of inactivity, extended period of inactivity, a period of failing to meet the requirements of your role, and/or a violation of the Code of Conduct. This process is important because it protects the community and its deliverables while also opens up opportunities for new contributors to step in.
```